app/src/components/card/cardImage.tsx

```diff
@@ -35,12 +37,14 @@ export const CardImage = ({ record, imageHeight }: CardImageProps) => {
         }}
         width={initialImageHeight * 2}
         height={initialImageHeight}
+        decoding="sync"
```

Comment:
Have you noticed if this prop has made any impact?

Reply:
It has not that I've noticed (was switching it between "sync" and "async" for a while during experimentation)

Reply:
But i think theoretically we do want it to be "sync"